docs/book/src/roadmap.md

```diff
@@ -2,56 +2,51 @@
-- [Integration with cloud provider VM scaling groups (MachinePools)](https://github.com/kubernetes-sigs/cluster-api/blob/bf51a2502f9007b531f6a9a2c1a4eae1586fb8ca/docs/proposals/20190919-machinepool-api.md)
-- [Machine failure remediation](https://github.com/kubernetes-sigs/cluster-api/blob/bf51a2502f9007b531f6a9a2c1a4eae1586fb8ca/docs/proposals/20191030-machine-health-checking.md)
-- [Provider end to end testing framework](https://github.com/kubernetes-sigs/cluster-api/blob/bf51a2502f9007b531f6a9a2c1a4eae1586fb8ca/docs/proposals/20191016-e2e-test-framework.md)
-
-## v0.3.x (v1alpha3+) ~ June/July 2020
-
-- [E2E Test plan](https://docs.google.com/spreadsheets/d/1uB3DyacOLctRjbI6ov7mVoRb6PnM4ktTABxBygt5sKI/edit#gid=0)
-- [Enable webhooks in integration tests](https://github.com/kubernetes-sigs/cluster-api/issues/2788)
-- [KCP robustness](https://github.com/kubernetes-sigs/cluster-api/issues/2753)
-- [KCP adoption](https://github.com/kubernetes-sigs/cluster-api/issues/2214)
-- [Clusterctl templating](https://github.com/kubernetes-sigs/cluster-api/issues/2339)
-- [Clusterctl cert-manager lifecycle](https://github.com/kubernetes-sigs/cluster-api/issues/2635)
-- [Post apply YAML experiment](https://github.com/kubernetes-sigs/cluster-api/issues/2395)
-- [MHC Support for control-plane nodes](https://github.com/kubernetes-sigs/cluster-api/issues/2836)
-- [Remote workload cluster watcher](https://github.com/kubernetes-sigs/cluster-api/issues/2414)
-- [Improved status conditions](https://github.com/kubernetes-sigs/cluster-api/issues/1658)
-    - UX: Clusterctl should offer a command to see the status of a cluster (library + cli).
-- [Spot instances support](https://github.com/kubernetes-sigs/cluster-api/issues/1876)
-- [Machine bootstrap failure detection](https://github.com/kubernetes-sigs/cluster-api/issues/2554)
-- [Machine load balancers](https://github.com/kubernetes-sigs/cluster-api/issues/1250)
-- [Extensible machine pre-delete hooks](https://github.com/kubernetes-sigs/cluster-api/issues/1514)
-- [Autoscaler integration](https://github.com/kubernetes-sigs/cluster-api/pull/2530)
+
+## v0.3.7 (v1alpha3+) ~ June/July 2020
+
+|Area|Description|Issue/Proposal|
+|--|--|--|
+|Testing|E2E Test plan|[Spreadsheet](https://docs.google.com/spreadsheets/d/1uB3DyacOLctRjbI6ov7mVoRb6PnM4ktTABxBygt5sKI/edit#gid=0)
+|Testing|Enable webhooks in integration tests|[#2788](https://github.com/kubernetes-sigs/cluster-api/issues/2788)|
+|Control Plane|KubeadmControlPlane robustness|[#2753](https://github.com/kubernetes-sigs/cluster-api/issues/2753)|
+|Control Plane|KubeadmControlPlane adoption|[#2214](https://github.com/kubernetes-sigs/cluster-api/issues/2214)|
+|Extensibility|Clusterctl library should support extensible templating|[#2339](https://github.com/kubernetes-sigs/cluster-api/issues/2339)|
```

Comment:
```suggestion
|Extensibility|Clusterctl library should support extensible templating|[#3052](https://github.com/kubernetes-sigs/cluster-api/pull/3052)|
```

Reply:
Everything else, apart from the spreadsheet points to issues so I kept it consistent

Reply:
@wfernandes what do you think here?

Reply:
I'm +1 to refer to the issue (you can get the PR from there)

Reply:
Yup. +1 I'm ok with it.